angelsindustries/prototypes/overrides/industries-override-functions.lua

```diff
@@ -570,17 +570,20 @@ end
 
 --REPLACE CONSTRUCTION ELECTRONIC BLOCKS
 function replace_con_mats(buildings)
-  for assembly_check in pairs(data.raw[buildings]) do
+  for assembly_check, build in pairs(data.raw[buildings]) do
     if data.raw.recipe[assembly_check] then
       local rec_check = data.raw.recipe[assembly_check]
       if rec_check.normal then
         ing_list = rec_check.normal.ingredients
         replace_blocks_list(ing_list)
+        data.raw[build.type][assembly_check].minable.results=ing_list
         ing_list = rec_check.expensive.ingredients
         replace_blocks_list(ing_list)
       else
         ing_list = rec_check.ingredients
+        log(serpent.block(ing_list))
```

Comment:
this should be removed, just noticing this here
```suggestion
```

Reply:
whoops